doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,291 @@
+
+	mpirun -np X gls_navier_stokes_2d naca.prm
+
+with X the number of processor used to run it.
+	
+However, it is highly recommended to launch the simulation on a supercomputer. To launch on a desktop machine, the ``time end`` can be set to ``3.0`` to see the beginning of the simulation. However, to get relevant results about the forces, it is better to simulate at least for 10 seconds so that a pseudo-steady regime settles.
+
+Results and discussion
+-----------------------
+
+The following average pressure and velocity fields are obtained for an angle of attack :math:`\alpha` such that :math:`\alpha \in \{0,5,7,9,11,15\}` : 
+
+.. image:: image/average_pressure.png
+
+.. image:: image/average_velocity.png
+
+
+It is already noticeable that the higher the angle of attack, the greater the pressure gradient. Following this observation, the lift coefficient :math:`C_L` is expected to increase with the angle of attack, until stall is reached. The variation of the lift and drag coefficients are given below with a comparison to the work of Kouser et al. `[1] <https://doi.org/10.1177/17568293211055656>`_. Both coefficients are computed using the following formula : 
+
+.. math::
+        C_L = \frac{F_L}{0.5\rho_{\infty} u_{\infty}^2 S} \; \; \; \; \; C_D = \frac{F_D}{0.5\rho_{\infty} u_{\infty}^2 S}
+        
+with :math:`F_L` and :math:`F_D` respectively the lift and drag forces. Those forces can be obtained in the ``force.00.dat`` and post processed using the ``post_processing.py`` python file included in the folder of this example. :math:`S` represents a reference area, here it is equal to the product of the chord length :math:`C` (equal to 1 in this example) multiplied by a unitary transversal length.
+        
+.. image:: image/cl_cd_results_plot.png
+
+There is a slight variation of the results obtained compared to the literature results. This may be due to the discontinuity at the trailing edge, or a resolution not precise enough. Still, the evolution observed is relevant.
+
+One can also see the low velocity zones on the upper part of the airfoil which corresponds to the recirculating zone: the ``noslip`` condition on the NACA imposes a zero velocity condition on the fluid. The following streamline representation helps to see the movements of the fluid inside the recirculating zone : 
+
+.. image:: image/naca_streamline_angles.png
+
+It can be observed that zones of recirculation form on the airfoil. This is due to two phenomena : first the flow outside of the boundary layer tends to "pull" it in its direction and the ``noslip`` boundary condition slows the fluid, then a positive pressure gradient, commonly referred to as adverse pressure gradient, on the upper surface pushes the fluid backwards. Following this, the boundary layer separates and a recirculation zone is formed. Below is represented the mean pressure coefficient :math:`C_p` on the airfoil with comparison to the literature. It is computed using the following formula : 
```

Comment:
```suggestion
It can be observed that zones of recirculation form on the airfoil. This is due to two phenomena : first the flow outside of the boundary layer tends to "pull" it in its direction, and the ``noslip`` boundary condition slows the fluid, then a positive pressure gradient, commonly referred to as adverse pressure gradient, on the upper surface pushes the fluid backward. Following this, the boundary layer separates, and a recirculation zone is formed. Below is represented the mean pressure coefficient :math:`C_p` on the airfoil with a comparison to the literature. It is computed using the following formula : 
```